hnn_core/network.py

```diff
@@ -633,7 +652,7 @@ def _attach_drive(self, name, drive, weights_ampa, weights_nmda, location,
 
     def _create_drive_conns(self, target_populations, weights_by_receptor,
                             location, space_constant, synaptic_delays,
-                            cell_specific=True):
```

Comment:
Is this still WIP? Since I don't see where `numsources` is used in this method?

Reply:
> Is this still WIP? Since I don't see where `numsources` is used in this method?

Correct, there's still a lot left to do.